src/views/Accessibility/AccessibiltyRequest/Documents/index.tsx

```diff
@@ -50,12 +51,17 @@ const AccessibilityDocumentsList = ({
         Cell: ({ row }: any) => (
           <>
             <UswdsLink
+              aria-describedby={`doc-link-${row.original.id}`}
+              target="_blank"
               asCustom={Link}
-              to={`/some-508-request/${row.original.name}`}
+              to={row.original.url}
             >
               {t('documentTable.view')}
+              <span className="usa-sr-only">document type</span>
```

Comment:
Might be worth leaving a to-do here to replace this with `row.original.document_type` once we have that implemented? Let's make sure that happens in the next story.